src/lib/block_producer/block_producer.ml

```diff
@@ -365,9 +365,8 @@ let handle_block_production_errors ~logger ~rejected_blocks_logger
         ; ("prev_state", Protocol_state.value_to_yojson previous_protocol_state)
         ; ("prev_state_proof", Proof.to_yojson previous_protocol_state_proof)
         ; ("next_state", Protocol_state.value_to_yojson protocol_state)
-          (* Commented out because for large blocks it's an oversized log *)
-          (* ; ( "internal_transition"
-             , Internal_transition.to_yojson internal_transition ) *)
+        ; ( "internal_transition"
```

Comment:
This change was probably made by accident

Reply:
I reverted this change